### ISSUE TITLE
Jejunitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6195]
**Preferred term label:**
Jejunitis

**Synonyms**
Inflammation of the jejunum

**Definition (free text, please give PubMed ID)**
Inflammation of the jejunum.
PMID: 26420973

**Parent term (use hpo.jax.org/app)**
Abnormality of the jejunum

**Comment by pnrobinson** [comment]
added as HP:0033143
